server/src/observability/metrics.ts

```
@@ -119,6 +119,8 @@ export {
   narrativeSimulationDurationSeconds
 };
 
+export default metrics;
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)